### PR TITLE
add node for visualization of script server navigation goals

### PR DESCRIPTION
**Pull request by floweisshardt**

### CODE REVIEW

File: cob_script_server/src/visualize_navigation_goals.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+        self.pubGoals.publish(markerarray)
+
+if __name__ == "__main__":
+    rospy.init_node('navigation_goal_visualizer')
+    p = VisualizerNavigationGoals()
+    r = rospy.Rate(1)
+    while not rospy.is_shutdown():
+        p.pubMarker()
+        r.sleep()
```
Comment (fmessmer) [inline cob_script_server/src/visualize_navigation_goals.py]: use the exception-aware [implementation](https://github.com/ipa320/cob_command_tools/blob/indigo_dev/cob_helper_tools/scripts/auto_init.py#L53-L56)

**Comment by fmessmer** [comment]
install tags missing!

Is `cob_script_server` the best package for this script?
what about `cob_helper_tools`?

could the script be made configurable to also visualize e.g. object_locations?

could you add an issue to MSH for the `csv_category_vis` stuff we just talked about?

File: cob_script_server/src/visualize_navigation_goals.py (hunk 1 above)
Comment (floweisshardt) [reply cob_script_server/src/visualize_navigation_goals.py]: done in https://github.com/ipa320/cob_command_tools/pull/191/commits/02714dcbcb2ac35b1fdce977be9ecbfc23b27c2d

**Comment by floweisshardt** [comment]
agreed to move to cob_helper_tools, done in https://github.com/ipa320/cob_command_tools/pull/191/commits/02714dcbcb2ac35b1fdce977be9ecbfc23b27c2d

I'll not implement object locations for the moment but this can be added later on.

**Comment by fmessmer** [comment]
- install tags
 - exception-aware `sleep()`

**Comment by floweisshardt** [comment]
install tags and exception-aware sleep is done in https://github.com/ipa320/cob_command_tools/commit/02714dcbcb2ac35b1fdce977be9ecbfc23b27c2d

**Review by fmessmer — APPROVED** [review]

**Pull request merged**